tests/ptf_runner.py

```diff
@@ -32,13 +32,32 @@ def ptf_collect(host, log_file, skip_pcap=False):
         allure.attach.file(filename_pcap, 'ptf_pcap: ' + filename_pcap, allure.attachment_type.PCAP)
 
 
+def get_dut_type(host):
+    dut_type_exists = host.stat(path="/sonic/dut_type.txt")
```

Comment:
This variable indeed holds "stat" of file `/sonic/dut_type.txt`. It means more than just "exists". Maybe `dut_type_stat` is a better variable name?

Reply:
Agree